doc/running-from-source.md

```diff
@@ -1,5 +1,8 @@
 # Setting up to run from source
 
+If you would like a longer, set-by-step set of instructions that doesn't
+assume much prior programming experience, see [this doc](./onboarding.md).
+
```

Comment:
typo: set-by-step should be step-by-step. (Would have fixed it myself, but with all the other comments, figured you would include it in the next commit.)

Reply:
Fixed in https://github.com/numberscope/frontscope/pull/148/commits/59fd25623bc1f78bae3e0e097eb8fbecb53e6926.